aten/src/ATen/cpu/vec/vec256/vec256_int.h

```diff
@@ -1522,6 +1522,41 @@ Vectorized<uint8_t> inline operator>>(const Vectorized<uint8_t>& a, const Vector
   return shift_256_8<false>(a, b);
 }
 
+#if defined(CPU_CAPABILITY_AVX2) && !defined(_MSC_VER)
+
+inline Vectorized<float> load_uint8_as_float(const uint8_t* src_data) {
+  // Load 8*uint8
+  __m256i input_256 = Vectorized<uint8_t>::loadu(src_data, 8);
```

Comment:
The `loadu` with element number introduces additional temporary buffer. It might be better to use for-loop directly. Please check the performance.

Reply:
Thanks for the comment and changed. There is a intrinsic can implement it without using the `loadu`  of element numbers.